### PR TITLE
Social: Update readme.txt to get rid of sharing limits

### PR DESCRIPTION
**Pull request by manzoorwanijk**
## Proposed changes:

* Remove the mention of share limits from readme for Social

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Confirm that there is no mention of sharing limits in readme.

### CODE REVIEW

File: projects/plugins/social/readme.txt [hunk 1]
```diff
@@ -29,11 +27,11 @@ Jetpack Social has scheduling tools that allow you to set your posts to publish
 
 **Social Image Generator**
 
-With the Jetpack Social Advanced plan, you can have engaging imagery created automatically using the Social Image Generator feature. You can overlay custom text onto your images and choose from a variety of styles to increase engagement on your social posts. Most importantly, you’ll save time by doing it all within the WordPress editor.
+With the Jetpack Social paid plan, you can have engaging imagery created automatically using the Social Image Generator feature. You can overlay custom text onto your images and choose from a variety of styles to increase engagement on your social posts. Most importantly, you’ll save time by doing it all within the WordPress editor.
```
Comment (spsiddarthan) [inline projects/plugins/social/readme.txt]: With the Jetpack Social plan, you can have engaging imagery created automatically using the Social Image Generator feature. You can overlay custom text onto your images and choose from a variety of styles to increase engagement on your social posts. Most importantly, you’ll save time by doing it all within the WordPress editor.
```suggestion
With the Jetpack Social plan, you can have engaging imagery created automatically using the Social Image Generator feature. You can overlay custom text onto your images and choose from a variety of styles to increase engagement on your social posts. Most importantly, you’ll save time by doing it all within the WordPress editor.
```

File: projects/plugins/social/readme.txt [hunk 2]
```diff
@@ -29,11 +27,11 @@ Jetpack Social has scheduling tools that allow you to set your posts to publish
...
 
 **Organic Content Sharing**
 
-Boost your social media engagement with Jetpack Social's Organic Content Sharing feature. Research indicates that manually published posts receive 50% more interaction compared to shared links. Our feature allows you to select custom images, videos, and text to share your blog posts as native social media content, enhancing engagement and saving you time. This feature is available with the Jetpack Social Advanced plan only.
+Boost your social media engagement with Jetpack Social's Organic Content Sharing feature. Research indicates that manually published posts receive 50% more interaction compared to shared links. Our feature allows you to select custom images, videos, and text to share your blog posts as native social media content, enhancing engagement and saving you time. This feature is available with the Jetpack Social paid plan only.
```
Comment (spsiddarthan) [inline projects/plugins/social/readme.txt]:
```suggestion
Boost your social media engagement with Jetpack Social's Organic Content Sharing feature. Research indicates that manually published posts receive 50% more interaction compared to shared links. Our feature allows you to select custom images, videos, and text to share your blog posts as native social media content, enhancing engagement and saving you time. This feature is available with the Jetpack Social plan.
```

File: projects/plugins/social/readme.txt (hunk 1 above)
Comment (manzoorwanijk) [reply projects/plugins/social/readme.txt]: Done

File: projects/plugins/social/readme.txt (hunk 2 above)
Comment (manzoorwanijk) [reply projects/plugins/social/readme.txt]: Done

**Review by spsiddarthan — DISMISSED** [review]

**Review by gmjuhasz — APPROVED** [review]

**Review by spsiddarthan — APPROVED** [review]

**Pull request merged**